contrib/linux/tests/test_single_file_tail.py

```diff
@@ -0,0 +1,603 @@
+#!/usr/bin/env python
+
+# Copyright 2020 The StackStorm Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+import logging
+import os
+import pathlib
+import time
+
+from watchdog.observers import Observer
+
+from file_watch_sensor import SingleFileTail
+
+WAIT_TIME = 1
+
+logger = logging.getLogger(__name__)
+
+
+def test_read_chunk_over_multibyte_character_boundary():
```

Comment:
Any reason why you didn't use unittest base class + assert method like we do in most other places?

Reply:
EDIT: I guess that may be because you can also run test directly as a script. Although, this should also b e possible with unittest